test/ts/src/testRpcCombinations.ts

```diff
@@ -54,7 +55,8 @@ export function runWithHttp1AndHttp2(cb: (config: TestConfig) => void) {
 
 export function runWithSupportedTransports(cb: (transport: grpc.TransportConstructor | undefined) => void) {
   const transports: {[key: string]: grpc.TransportConstructor | undefined} = {
-    "defaultTransport": undefined
+    "defaultTransport": undefined,
+    "fetchWithOptions": fetchRequestWithOptions({}),
```

Comment:
I assume we may need to add this key to some test cases as well?

Reply:
This seems to be used by all test cases as a way to provide transport - so it sohlud do the trick. Testing setup is rather complictaed, but seems to be prepared for testing custom transports like this. Would like a comment from @MarcusLongmuir on this.